infra/bisector.py

```diff
@@ -139,9 +147,10 @@ def bisect(old_commit, new_commit, test_case_path, fuzz_target, build_data):
 
     while old_idx - new_idx > 1:
       curr_idx = (old_idx + new_idx) // 2
+      logging.info('Testing against %s', commit_list[curr_idx])
```

Comment:
nit suggestion: might be useful to print indexes in the commits list as well, just in case the bisection logic appears to be buggy

Reply:
Done!